OpmlImportFromPathActivity: onActivityResult NPE

Version: 1.6.0.9

```
java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=-1, data=Intent { flg=0x80000 (has extras) }} to activity {de.danoeh.antennapod/de.danoeh.antennapod.activity.OpmlImportFromPathActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.Uri.toString()' on a null object reference
	at android.app.ActivityThread.deliverResults(ActivityThread.java:3659)
	at android.app.ActivityThread.handleSendResult(ActivityThread.java:3702)
	at android.app.ActivityThread.access$1300(ActivityThread.java:155)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1366)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5343)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.Uri.toString()' on a null object reference
	at de.danoeh.antennapod.activity.OpmlImportFromPathActivity.onActivityResult(OpmlImportFromPathActivity.java:138)
	at android.app.Activity.dispatchActivityResult(Activity.java:6218)
	at android.app.ActivityThread.deliverResults(ActivityThread.java:3655)
	... 10 more
```

--- a/app/src/main/java/de/danoeh/antennapod/activity/OpmlImportFromPathActivity.java
+++ b/app/src/main/java/de/danoeh/antennapod/activity/OpmlImportFromPathActivity.java
@@ -135,7 +135,7 @@ public class OpmlImportFromPathActivity extends OpmlImportBaseActivity {
         super.onActivityResult(requestCode, resultCode, data);
         if (resultCode == RESULT_OK && requestCode == CHOOSE_OPML_FILE) {
             Uri uri = data.getData();
-            if(uri.toString().startsWith("/")) {
+            if(uri != null && uri.toString().startsWith("/")) {
                 uri = Uri.parse("file://" + uri.toString());
             }
             importUri(uri);

